python/ray/serve/tests/test_deploy.py

```diff
@@ -12,6 +12,8 @@
 from ray import serve
 from ray.serve.utils import get_random_letters
 
+TEST_FILE_PATH = "/tmp/test_deploy.txt"
```

Comment:
I don't know if `/tmp` is accessible on all CI versions (e.g., in windows) and this might not get cleaned up between runs on the CI machine. Better to use the `tempfile` module.

Reply:
ah that sounds cleaner indeed